.eslintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "plugins": ["react", "react-hooks", "import"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "rules": {
+    "prettier/prettier": [
+      2,
+      {
+        "singleQuote": true,
+        "jsxSingleQuote": true
+      }
+    ],
+    "jsx-quotes": [2, "prefer-single"],
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Did you meant to turn it off? (Or am I misunderstanding how this works?)

Reply:
yes, I think its fine to have the any type and no-unused-vars we should revisit. We should actually revisit this file in general, i'm mostly mainly setting things up